qdrant-landing/content/documentation/hybrid-cloud/platform-deployment-options.md

```diff
@@ -5,34 +5,136 @@ weight: 4
+
+### More on Amazon Elastic Kubernetes Service
+
+- [Getting Started with Amazon EKS](https://docs.aws.amazon.com/eks/)
+- [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html)
+- [Amazon EKS API Reference](https://docs.aws.amazon.com/eks/latest/APIReference/Welcome.html)
+
+![Digital Ocean](/documentation/cloud/cloud-providers/digital-ocean.jpg)
+
+## Digital Ocean
+
+DigitalOcean Kubernetes (DOKS) is a managed Kubernetes service that lets you deploy Kubernetes clusters without the complexities of handling the control plane and containerized infrastructure. Clusters are compatible with standard Kubernetes toolchains and integrate natively with DigitalOcean Load Balancers and volumes.
+
+### More on DigitalOcean Kubernetes
+
+- [Getting Started with DOKS](https://docs.digitalocean.com/products/kubernetes/getting-started/quickstart/)
+- [DOKS - How To Guides](https://docs.digitalocean.com/products/kubernetes/how-to/)
+- [DOKS - Reference Manual](https://docs.digitalocean.com/products/kubernetes/reference/)
+
+![Google Cloud Platform](/documentation/cloud/cloud-providers/gcp.jpg)
+
+## Google Cloud Platform (GCP)
+
+[Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) is a managed Kubernetes service that you can use to deploy and operate containerized applications at scale using Google's infrastructure. GKE provides the operational power of Kubernetes while managing many of the underlying components, such as the control plane and nodes, for you.
+
+### More on the Google Kubernetes Engine
+
+- [Getting Started with GKE](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+- [GKE Tutorials](https://cloud.google.com/kubernetes-engine/docs/tutorials)
+- [GKE Documentation](https://cloud.google.com/kubernetes-engine/docs/)
+
+![Microsoft Azure](/documentation/cloud/cloud-providers/azure.jpg)
+
+## Mircrosoft Azure
+
+With [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-in/products/kubernetes-service), you can start developing and deploying cloud-native apps in Azure, datacenters, or at the edge. Get unified management and governance for on-premises, edge, and multicloud Kubernetes clusters. Interoperate with Azure security, identity, cost management, and migration services.
+
+### More on Azure Kubernetes Service
+
+- [Getting Started with AKS](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks-start-here)
+- [AKS Documentation](https://learn.microsoft.com/en-in/azure/aks/)
+- [Best Practices with AKS](https://learn.microsoft.com/en-in/azure/aks/best-practices)
+
+![Oracle Cloud Infrastructure](/documentation/cloud/cloud-providers/oracle.jpg)
 
 ## Oracle Cloud Infrastructure
```

Comment:
Oracle does not seem to set allowVolumeExpansion to true either by default: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV.htm#:~:text=The%20cluster's%20default%20storage%20class%20is%20initially%20set%20according%20to,initially%20set%20as%20the%20default.

You also have to install the snapshot controller and snapshot class manually: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV.htm#contengcreatingpersistentvolumeclaim_topic-Provisioning_PVCs_on_BV-PV_From_Snapshot_CSI__section_volume-snapshot-prerequisites